pvlib/tests/iotools/test_pvgis.py

```diff
@@ -509,6 +519,12 @@ def test_get_pvgis_map_variables(pvgis_tmy_mapped_columns):
     assert all([c in pvgis_tmy_mapped_columns for c in actual.columns])
 
 
+@pytest.mark.remote_data
+def test_read_pvgis_horizon():
+    df = get_pvgis_horizon(35.171051, -106.465158)
+    elv = df.horizon_angles
+    assert_series_equal(elv, data_horizon_abq)
```

Comment:
I would probably recommend constructing the full dataframe returned and comparing that. Right now the `horizon_azimuth` column seems untested and it doesn't seem like it would require much more effort to compare a dataframe instead of a series.

Reply:
right, that's probably worth doing. to avoid further clutter of `test_pvgis.py` I saved the dataframe as `test_read_pvgis_horizon.csv` in the data directory, and then did an `assert_frame_equal` .